tools/gulp/tasks/lint.ts

```diff
@@ -46,6 +54,56 @@ task('madge', ['material:clean-build'], () => {
   });
 });
 
+task('ownerslint', () => {
+  let errors = 0;
+
+  let ownedPaths = readFileSync(ownersFilePath, 'utf8').split('\n')
+      // Trim lines.
+      .map(line => line.trim())
+      // Remove empty lines and comments.
+      .filter(line => line && !line.startsWith('#'))
+      // Split off just the path glob and turn it into a regex.
+      .map(line => new Minimatch(line.split(/\s+/)[0], {dot: true, matchBase: true}))
+      // Add in the paths we're ignoring.
+      .concat(ignoreOwners.map(path => new Minimatch(path, {dot: true, matchBase: true})));
+
+  for (let paths = getChildPaths('.'); paths.length;) {
+    paths = Array.prototype.concat(...paths
```

Comment:
You should be able to get rid of the `Array.prototype.concat` using a spread: `paths = [...paths.filter]`.

Reply:
I think thats the equivalent of `slice(...paths)` not `concat(...paths)`